cli/src/commands/merge/index.ts

```diff
@@ -0,0 +1,36 @@
+import { getCurrentBranchDetails, Schemas } from '@xata.io/client';
+import fetch from 'node-fetch';
+import { BaseCommand } from '../../base.js';
+import deepmerge from 'deepmerge';
+import Codegen from '../codegen/index.js';
+
+export default class Merge extends BaseCommand {
+  static description = 'Merge the current branch with another branch';
+
+  static examples = [];
+
+  static flags = {};
+
+  static args = [{ name: 'branch', description: 'The branch to merge the current branch with', required: true }];
```

Comment:
I'd probably add a way to ``xata merge foo-a foo-b``, not sure about the order (the second one is optional of course).

Also maybe an interactive way could be great: ``xata merge`` and it asks both branches origin/destination. Maybe destination with feault to ``main`` branch.

This can go to the improvements card.